[IAM-5206] SSO: multiple Sauce Labs organizations with single identity provider (#1793)
saucelabs/sauce-docs: docs/basics/sso/setting-up-sso.md

- An enterprise license for Sauce Labs
- Organization admin access (see [User Roles](/basics/acct-team-mgmt/managing-user-info) for more information)

> ## Special cases

Reviewer: Do you expect to add more exceptions that require additional steps for SAML SSO integration? 
```suggestion
## Special Cases
```

Author: not for now, but obviously we will add another cases in the future

Reviewer: Set this up so that is works for now, and you can add more cases as needed.

> If your SSO setup falls into one of these special cases, refer to these setup steps:

Reviewer: ```suggestion
```

Author: Do you want me to remove this line completely?

Reviewer: Yes, we can add a intro sentence later when you add more cases.

> If your SSO setup falls into on of the special cases, please first refer to these setup steps:

Reviewer: ```suggestion
If your SSO setup falls into one of these special cases, refer to these setup steps:
```

Author: fixed

> - My company has more than one organization at Sauce Labs to be integrated with SAML SSO and my Identity Provider requires a unique entity ID for Service Providers. -> [Setup steps](/basics/sso/setting-up-sso-special-cases)

Reviewer: ```suggestion
- If your company has more than one organization at Sauce Labs to be integrated with SAML SSO and your Identity Provider requires a unique entity ID for Service Providers, [see these special setup steps](/basics/sso/setting-up-sso-special-cases)
```

Author: fixed

> - My company has **more than one organization at Sauce Labs** which need to be integrated with SAML SSO and my Identity Provider **requires unique entity ID of Service Providers.** -> [Setup steps](/basics/sso/setting-up-sso-special-cases)

Reviewer: ```suggestion
- My company has more than one organization at Sauce Labs to be integrated with SAML SSO and my Identity Provider requires a unique entity ID for Service Providers. -> [Setup steps](/basics/sso/setting-up-sso-special-cases)
```

Author: fixed
